pkg/controller/workload/job/job_controller.go

```diff
@@ -386,6 +388,16 @@ func ConstructWorkloadFor(job *batchv1.Job, scheme *runtime.Scheme) (*kueue.Queu
 			QueueName: queueName(job),
 		},
 	}
+
+	priorityClassName := job.Spec.Template.Spec.PriorityClassName
```

Comment:
Do we populate the priorityClassName from the default for pods too? If so, we should do the same here.

Reply:
yes. Will add it later.

Reply:
Done